software/wordclock/src/ClockFace.cpp

```diff
@@ -274,6 +291,16 @@ bool FrenchClockFace::stateForTime(int hour, int minute, int second, bool show_a
 
   switch (leftover)
   {
+#ifdef NODO
+  case 4:
+    _state[113] = true;
```

Comment:
Why not do this in the mapminute() method, then the change only required at single point?

Reply:
You're right, I'll do that.